docs/MULTIPASS.md

```diff
@@ -37,7 +37,7 @@ It took me about 2-3 minutes to run through everything after installing multipas
     This command will update the key with your local public key value and start the VM.
 
     ```sh
-    sed "s/ssh-rsa.*/$(cat $HOME/.ssh/id_*.pub)/" cloud-config.txt | multipass launch --name faasd --cloud-init -
+    sed "s/ssh-rsa.*/$(cat $HOME/.ssh/id_*.pub | sed 's/\//\\\//g')/" cloud-config.txt | multipass launch --name faasd --cloud-init -
```

Comment:
This ends up being much more complicated.

Can you share the sample we need to use for a repro?

Perhaps there's an alternative syntax or approach for making this replacement that we can think of?

Alex